queries.sql

```diff
@@ -1,3 +1,46 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered = TRUE;
+SELECT * FROM animals WHERE name <> 'Gabumon';
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT after_delete;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO after_delete;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+
+-- 
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT neutered, AVG(escape_attempts) AS avg_escape_attempts 
+FROM animals 
+GROUP BY neutered 
+ORDER BY avg_escape_attempts DESC 
+LIMIT 1;
+
+SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight 
+FROM animals 
+GROUP BY species;
+
+
+SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight 
+FROM animals 
+GROUP BY species;
+
+
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts 
+FROM animals 
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' 
+GROUP BY species;
+
```

Comment:
- [ ] Please 🙏🏽  **Take a screenshot of the results of your queries.** this is a project requirement to approve this milestone.